### PR TITLE
Add --print-path option and README updates

### PR DESCRIPTION
**Pull request by brunetton**
Thank you for this great software, I was looking for a tool like this one since some weeks ago.

I used it today and I missed some details to understand how it works. So I updated README and added a command line option to print signatures folder.

Hope this looks okay to you. Tell me !

Thanks again !

### CODE REVIEW

File: README.md
Comment (svenssonaxel) [inline README.md:38]: I wish the empty `""` wasn't needed.

File: README.md
Comment (svenssonaxel) [inline README.md:43]: Good, but the instruction under `## How` also needs correcting.

File: README.md
Comment (svenssonaxel) [inline README.md:44]: For clarity, perhaps add a line such as `Thereafter, you can begin using pdf-sign. For example:`
Otherwise the below code block could appear like it was about creating signatures.

File: pdf-sign
Comment (svenssonaxel) [inline pdf-sign:10]: This will print the correct path in almost all cases. The exception is when
* The user prefers to put the signatures under `~/.config` in order not to litter the home directory.
* `~/.config/pdf_signatures` does not exist (yet)
* `$PDF_SIGNATURE_DIR` and `XDG_CONFIG_HOME` are both unset.

In this situation, `pdf-sign "" --print-path` will print `/home/user/.pdf_signatures` even though it should have printed `/home/user/.config/pdf_signatures`.
I'm not exactly sure how to fix this, but I think it's a shame to make this improvement without going all the way.

File: pdf-sign
Comment (svenssonaxel) [inline pdf-sign:411]: Perhaps `-S` and `-s` are confusingly similar?

**Review by svenssonaxel — CHANGES_REQUESTED** [review]
Glad that you like the tool.
I think this is an improvement, but I think we should improve it some more before merging. You're very welcome to address these comments, but if you don't want to, I'll eventually get around to doing it myself. In the meantime, thank you!

File: README.md
Comment (brunetton) [reply README.md:38]: I wish too, it's quite ugly ! I'm not an argparse expert but I havn't succeed achieving it using argparse.

Anyway, after reconsideration I'd remove this option, as there is an error message that prints the signatures dir if it doesn't exists. In the README we could write something like:

> To know the signature directory to create, run `./pdf-sign my_pdf.pdf` and see the error message

What would you think about this ?

File: README.md
Comment (brunetton) [reply README.md:43]: Oh yes indeed. I'll merge both sections if you are okay

File: pdf-sign
Comment (brunetton) [reply pdf-sign:10]: Indeed. 

But this is out of the scope of this PR, as this is not a problem introduced by this PR, it was already the case before

Anyway, I'll finally suggest to turn back and forget adding this option.

**Comment by svenssonaxel** [comment]
@brunetton I came up with a different solution that requires no extra options. Essentially, the help text and error messages are customized to explain the situation on your particular system. Check my branch `dev` and let me know what you think.

**Comment by svenssonaxel** [comment]
@brunetton PR rejected since the solution I mentioned above is now merged. Feel free to open an issue if you are not satisfied.

**Pull request closed without merging**
